resources/views/pages/register.twig

```diff
@@ -0,0 +1,133 @@
+{% extends "layouts/app.twig" %}
+{% import 'macros/base.twig' as m %}
+{% import 'macros/form.twig' as f %}
+
+{% block title %}{{ __('Register') }}{% endblock %}
+
+{% block content %}
+    <div class="col-md-12">
+        <h1>{{ __('Register') }}</h1>
+        {{ __('By completing this form you\'re registering as a Chaos-Angel. This script will create you an account in the angel task scheduler.') }}
+
+        {{ msg() }}
+        {% for message in errors|default([]) %}
+            {{ m.alert(__(message), 'danger') }}
+        {% endfor %}
+
+        <form action="" enctype="multipart/form-data" method="post">
+            {{ csrf() }}
+            <div class="row">
+                <div class="col-md-6">
+                    <div class="row">
+                        <div class="col-sm-4">
+                            {{ f.input('login', __('Nick'), 'text', true) }}
+                            <span class="help-block">
+                                {{ m.glyphicon('info-sign') }}
+                                {{ __('Use up to 23 letters, numbers, connecting punctuations or spaces for your nickname.') }}
```

Comment:
One point of moving stuff to templates is to change the texts to use an abbreviation text like `[page_name].[part]` so that not all translations have to be updated only because the English text changes.
And another point is that (more or less changeable) numbers/names should be generalized to use placeholders like `{{ __('The user has %u bananas!', count(user.bananas)) }}` so that refactoring is less painful.